src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js

```diff
@@ -187,6 +187,7 @@ export const AzureComputeModalBase = ({
   // Helper functions -- begin
   const applyChanges = _.flow(
     Utils.withBusyState(setLoading),
+    // If you update this text, update `run-analysis-azure`
```

Comment:
in these comments, can we specify ... in integration-tests

Reply:
at some point this would be solved by i18n when we have text strings externalized and likely pull-able from a package that the main app and the integration tests can import.... but that may be a ways off.